scripts/cli/src/util/checkBridgeBalance.ts

```diff
@@ -0,0 +1,164 @@
+import { ApiSingleton } from "../util/init";
+import PrettyError from "pretty-error";
+import * as prom from "./promClient.js";
+import express from "express";
+
+/// Express
+const app = express();
+const port = 8888;
+
+async function main(): Promise<void> {
+	type userInfo = {
+		polyx: number;
+		lastTimeBridged: number;
+	};
+	prom.injectMetricsRoute(app);
+	prom.startCollection();
+	console.log(
+		`Starting events monitor watching endpoint: ${process.env.WS_PROVIDER}`
+	);
+	app.listen(port, () =>
+		console.log(`Offences monitor running on port ${port}`)
+	);
+	const api = await ApiSingleton.getInstance();
+	const BRIDGE_LIMIT = 500_000_000;
+	const BRIDGE_POLY_TOTAL = 1_000_000_000;
+	const BRIDGE_TIME_PERIOD = await blocksToSeconds();
+	let userArray: Map<string, userInfo> = new Map();
```

Comment:
`userArray` is not an array any more.  I would recommend naming it `userTxs`.

Reply:
Alright would work on the changes.